src/main/java/org/broadinstitute/hellbender/tools/gvs/extract/ExtractCohort.java

```diff
@@ -87,11 +102,11 @@
     private boolean emitPLs = false;
 
     @Argument(
-            fullName = "vqslod-filter-genotypes",
-            doc = "Should VQSLOD filtering be applied at the genotype level",
+            fullName = "filter-type",
+            doc = "What type of filtering should be applied / at what level",
             optional = true
     )
-    private boolean performGenotypeVQSLODFiltering = true;
+    private FilteringType filteringType = FilteringType.NONE; // TODO do we want to make sure this is always / required?
```

Comment:
I think this should be FilteringType.GENOTYPE by default, and if they don't supply a filter_set we can either throw an error or change the value for this to be NONE (maybe the former)